packages/dds/map/src/test/mocha/map.fuzz.spec.ts

```diff
@@ -78,6 +83,10 @@ const reducer = combineReducers<Operation, State>({
 	deleteKey: ({ client }, { key }) => {
 		client.channel.delete(key);
 	},
+	useHandle: ({ random, client }, { handle }) => {
+		const keyNames = Array.from({ length: defaultOptions.keyPoolSize }, (_, i) => `${i}`);
```

Comment:
@Abe27342 curious if you have any ideas here.  the basic goals is that the harness pushes out a handle, and then the individual tests consume them. the problem here is that that the reducer is picking the key, which means it won't get captured in the output, so won't replay/minimize quite right.

Reply:
lets not block on this, but we'll probably want to fix it if we can